langchain/src/retrievers/parent_document.ts

```diff
@@ -210,7 +210,11 @@ export class ParentDocumentRetriever extends MultiVectorRetriever {
             metadata: { ...subDoc.metadata, [this.idKey]: parentDocId },
           })
       );
-      await this._storeDocuments(parentDoc, taggedSubDocs, addToDocstore);
+      await this._storeDocuments(
+        { parentDocId: parentDoc },
```

Comment:
Did you mean this?
```suggestion
        { [parentDocId]: parentDoc },
```

Without the `[]` it will make the key `parentDocId`. By wrapping it in brackets it'll use the actual value of the `parentDocId` variable.

Reply:
Thanks for the thorough review @bracesproul 🙏 
I guess there should really be some test for this 🙈

Reply:
Could you add a test for this method? Would help catch future issues.